### PR TITLE
[WIP] Permissions, tests and admin page

### PR DESCRIPTION
**Pull request by cherrybeans**
Need some help on how to update the commitee throught the api.

### CODE REVIEW

File: committee_admissions/admissions/serializers.py [hunk 1]
```diff
@@ -25,7 +25,16 @@ class Meta:
 class CommitteeSerializer(serializers.HyperlinkedModelSerializer):
     class Meta:
         model = Committee
-        fields = ('url', 'pk', 'name', 'description', 'response_label', 'logo', 'detail_link')
+        fields = ('url', 'pk', 'name', 'description', 'response_label', 'detail_link')
+
+    def create(self, validated_data):
```
Comment (cherrybeans) [inline committee_admissions/admissions/serializers.py]: This is never run, even though on POST it should.

File: frontend/src/routes/AdminPage/index.js [hunk 2]
```diff
@@ -164,3 +186,95 @@ class AdminPage extends Component {
...
+    const submission = {
+      name: committeeNames[committee],
+      description: values.description,
+      response_label: values.replyText
+    };
+
+    console.log(submission);
+
+    fetch(`${apiRoot}/api/committee/`, {
+      method: "POST",
```
Comment (cherrybeans) [inline frontend/src/routes/AdminPage/index.js]: Get the error `{"name":["committee` with this name already exists."]}`, so I need help on how to customize the api to allow updates. The way I thought should work based on previous success didn't.

File: committee_admissions/admissions/views.py [hunk 3]
```diff
@@ -48,39 +42,52 @@ def get_serializer_class(self):
...
+            return super().get_queryset().prefetch_related(
+                'committee_applications', 'committee_applications__committee'
+            )
+
+        group = Membership.objects.filter(user=user, role=constants.LEADER).first().abakus_group
+
+        qs = CommitteeApplication.objects.filter(committee__name=group.name
+                                                 ).select_related('committee')
+
+        return super().get_queryset().filter(committee_applications__committee__name=group.name
```
Comment (odinuge) [inline committee_admissions/admissions/views.py]: This looks like acid, but it does indeed work :100: 

Tests next! :1st_place_medal:

File: committee_admissions/admissions/views.py [hunk 4]
```diff
@@ -48,39 +42,52 @@ def get_serializer_class(self):
...
-
-class CommitteeApplicationViewSet(viewsets.ModelViewSet):
-    queryset = CommitteeApplication.objects.all()
-    serializer_class = CommitteeApplicationSerializer
-
-
-class UserViewSet(viewsets.ModelViewSet):
-    queryset = User.objects.all()
-    serializer_class = UserSerializer
+    @list_route(methods=['GET'])
```
Comment (odinuge) [inline committee_admissions/admissions/views.py]: We can use this endpoint, `/api/application/my`, to fetch the application

**Pull request merged**